backend/mechanismbackend/api/models.py

```diff
@@ -47,3 +47,47 @@ def clean(self):
                 not self.outputT3):
             raise ValidationError("Mechanism needs at least one input or output to be specified.")
 
+    def get_dissimilarity(self, other):
+        axes_dissimilarity = self._get_axes_dissimilarity(other)
+        name_dissimilarity = self._get_name_dissimilarity(other)
+        transmission_dissimilarity = self._get_transmission_dissimilarity(other)
+        return 0.5 * axes_dissimilarity + 0.35 * name_dissimilarity + 0.15 * transmission_dissimilarity
```

Comment:
According to @ThijsRoumen, we do not need to take name similarity into account anymore. Could you please remove it, @alpreu, and change the remaining weightings to 0.8 and 0.2?